tests/selenium/conftest.py

```diff
@@ -64,6 +63,29 @@ def double_click(self):
         ac.double_click()
         return ac.perform()
 
+    def exists(self):
```

Comment:
Lots of ported tests from this point forward and added capabilities in the table testing helper. Will not comment on individual test ports.